proto/ibc/core/client/v1/query.proto

```diff
@@ -205,3 +216,27 @@ message QueryUpgradedConsensusStateResponse {
   // Consensus state associated with the request identifier
   google.protobuf.Any upgraded_consensus_state = 1;
 }
+
+// QueryVerifyMembershipProofRequest is the request type for the Query/VerifyMembershipProof RPC method
+message QueryVerifyMembershipProofRequest {
+  // client unique identifier.
+  string client_id = 1;
+  // the proof to be verified by the client.
+  bytes proof = 2;
+  // the height of the commitment root at which the proof is verified.
+  ibc.core.client.v1.Height proof_height = 3 [(gogoproto.nullable) = false];
+  // the commitment key path.
+  ibc.core.commitment.v1.MerklePath merkle_path = 4 [(gogoproto.nullable) = false];
+  // the value which is proven.
+  bytes value = 5;
+  // optional time delay
+  uint64 time_delay = 6;
+  // optional block delay
+  uint64 block_delay = 7;
+}
+
+// QueryVerifyMembershipProofResponse is the response type for the Query/VerifyMembershipProof RPC method
+message QueryVerifyMembershipProofResponse {
+  // result indicating success or failure of proof verification.
+  bool result = 1;
```

Comment:
This could also be `success`, dropping the comment here if anyone feels one is better than the other. I feel indifferent

Reply:
I like `success`, not sure what a bool for `result` would mean

Reply:
```
if resp.Success {

}
```

vs 
```
if resp.Result {

}
```

Reply:
Aye, when its written like that I completely agree 👍🏻